autoscoper/src/ui/ui-files/AutoscoperMainWindow.ui

```diff
@@ -384,6 +384,49 @@ Dialog</string>
          </property>
         </widget>
        </item>
+       <item>
+        <widget class="Line" name="line_2">
+         <property name="orientation">
+          <enum>Qt::Vertical</enum>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="QToolButton" name="toolButtonPreviousCurveSet">
+         <property name="text">
+          <string>Previous
+Tracking
+Set (J)</string>
```

Comment:
Is the line wrapping required ?

Reply:
Unfortunately, it is:
```xml
          <string>Previous 
            Tracking
            Set (J)
          </string>
```
![image](https://github.com/user-attachments/assets/5a8e7c92-68b4-407a-a2a0-7b475a7aecb8)

Reply:
In that case, it should be possible to use html, see https://doc.qt.io/qt-5/richtext-html-subset.html

```
<center><p>Previous</p><p>Tracking</p><p>Set (J)</p></center>
```

Reply:
HTML is not supported in this case, instead using a line break should ensure the text is centered:
```
Previous\nTracking\nSet (J)
```

Reply:
Thanks for looking into this. After reviewing remaining of `AutoscoperMainWindow.ui`, proposed approach is consistent with how other `QToolButton` are setup